nre/bpf/nodefw/nodefw-ebpf/rust-toolchain.toml

```diff
@@ -0,0 +1,13 @@
+[toolchain]
+channel = "nightly"
```

Comment:
Why is nightly needed?

Reply:
imo, requiring nightly is a blocker for this going into production

Reply:
afaict, it's because they use build-std=core in the cargo command the generates the bpf linking.  I assume this is because they need to use the targeted system to include system specific stuff to get it to work since the pre-built core will be missing it.

https://doc.rust-lang.org/cargo/reference/unstable.html#build-std

we'll need to allow this for this crate if we are going to use it.

Reply:
Nightly as a channel is unstable and updates, well nightly. This means every day you roll the dice to see if things will be broken which is a nonstarter for anything we want to release into production.

What's the rationale for the multiple added workspaces? We have a lot of tooling that currently relies on the fact that we have a single workspace that everything is in and having many more will either mean more complex ci or that tests/builds will likely be unintentionally skipped in ci.

So I'd like to see:
- if we can merge this back into a single workspace
- can we rely on our already pinned stable rustc compiler version and use it with the RUSTC_BOOTSTRAP escape hatch mechanism or explicitly pin an exact version of the compiler to use for the purpose of building bpf programs

Reply:
we spoke in chat and i'll update the RUSTC_BOOSTRAP workaround you mentioned.  thanks!! 🥳